File: exercice1/python/Article.py
class Article:
	
	def __init__(self, ref, nom, price, quantite):
		self.__reference=ref
		self.__nom=nom
		self.__price=price
		self.__quantite=quantite
	def get_ref(self):
		return self.__reference
	def get_name(self):
		return self.__nom
	def get_price(self):
		return self.__price
	def getCurrentPrice(self):
		return str(self.__price)+'TND'
	def __str__(self):
		return self.__nom

File: exercice1/Article.php
<?php
namespace raouf;
class Article{
	private $nrf;
	private $nom;
	private $price;
	private $quantite;

	public function __construct($nrf, $nom, $prix, $quantite){
		$this->nrf=$nrf;
		$this->nom=$nom;
		$this->price=$prix;
		$this->quantite=$quantite;
	}
	public function toString(){
		
	}
	public function get_nrf(){
		return $this->nrf;
	}
	public function get_name(){
		return $this->nom;
	}
	public function get_price(){
		return $this->price;
	}
}
?>

